pkg/models/task_collection_test.go

```diff
@@ -1272,8 +1314,38 @@ func TestTaskCollection_ReadAll(t *testing.T) {
 			want: []*Task{
 				task1,
 				task2,
+				task3,
```

Comment:
Shouldn't this also depend on `include nulls`?

Reply:
Why? Tasks without labels fully comply with the condition "does not contain the specified label"

Reply:
That does make sense, I guess I was just confused by the change in assertions.